react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,74 @@
+import React from "react"
+
+const content = [
+  {
+    lang: "HTML",
+    text: "Hypertext Markup Language (HTML) is the standard markup language for creating web pages and web applications. With Cascading Style Sheets (CSS) and JavaScript, it forms a triad of cornerstone technologies for the World Wide Web."
+  },
+  {
+    lang: "CSS",
+    text: "Cascading Style Sheets (CSS) is a style sheet language used for describing the presentation of a document written in a markup language such as HTML.CSS is a cornerstone technology of the World Wide Web, alongside HTML and JavaScript."
+  },
+  {
+    lang: "JS",
+    text: "JavaScript, often abbreviated as JS, is a high-level, interpreted programming language that conforms to the ECMAScript specification. JavaScript has curly-bracket syntax, dynamic typing, prototype-based object-orientation, and first-class functions."
+  }
+]
+
+class Accordion extends React.Component {
+  constructor(props) {
+    super(props)
+    this.handleClick = this.handleClick.bind(this)
+    this.handleDbClick = this.handleDbClick.bind(this)
+  }
+
+  handleClick(event) {
+    const $textContent = document.querySelectorAll("[data-view]")
+    const id = event.target.getAttribute("id")
+
+    for (let i = 0; i < $textContent.length; i++) {
+      $textContent[i].getAttribute("id") === `${id}-content`
+        ? $textContent[i].className = "row"
+        : $textContent[i].className = "row hidden"
+    }
+  }
```

Comment:
`handleClick` should take a single parameter that is the id of an item then set that id into state.

```JSX
handleClick(itemId) {
  this.setState({ openId: itemId});
}
```